pkg/machine/qemu/machine.go

```diff
@@ -537,6 +645,45 @@ func (v *MachineVM) SSH(name string, opts machine.SSHOptions) error {
 	return cmd.Run()
 }
 
+func (v *MachineVM) sshConfig() string {
+	config := fmt.Sprintf("Host %s\n", v.Name)
+	config += fmt.Sprintf("    IdentityFile %s\n", v.IdentityPath)
+	config += fmt.Sprintf("    User %s\n", v.RemoteUsername)
+	config += fmt.Sprintf("    Hostname %s\n", "localhost")
+	config += fmt.Sprintf("    Port %d\n", v.Port)
+	config += fmt.Sprintf("    UserKnownHostsFile %s\n", "/dev/null")
+	config += fmt.Sprintf("    StrictHostKeyChecking %s\n", "no")
+	return config
+}
+
+func (v *MachineVM) SSHocker(name string, source, target string, readonly bool) error {
+	volumeDefinition := source + ":" + target
+	if readonly {
+		volumeDefinition += ":ro"
+	}
+
+	vmConfigDir, err := machine.GetConfDir(vmtype)
+	if err != nil {
+		return err
+	}
+	config := v.sshConfig()
+	sshConfigFile := filepath.Join(vmConfigDir, v.Name+".config")
+	err = ioutil.WriteFile(sshConfigFile, []byte(config), 0666)
+	if err != nil {
+		return err
+	}
+
+	sshDestination := fmt.Sprintf("%s:%d", v.Name, v.Port)
+
+	args := []string{"run", "-F", sshConfigFile, "-v", volumeDefinition,
+		sshDestination, "sleep", "36500d"} // "Törnrosa sov i 100 år"
```

Comment:
Engelska kommentarer till de få av oss som inte gör det svenska tack!

Reply:
Will change these in the other PR

Reply:
https://en.wikipedia.org/wiki/Sleeping_Beauty